doc/us/reference.html

```diff
@@ -113,6 +113,13 @@ <h3>Copas dispatcher main functions</h3>
         removing the server. If <code>keep_open</code> is <code>true</code>, the socket
         is removed from the scheduler but it is not closed.
     </dd>
+	
+	<dt><strong><code>copas.removethread(thread)</code></strong></dt>
+    <dd>Removes a thread added to the Copas scheduler. 
+    	Takes a <code>thread</code> created by <code>copas.addthread()</code> and removes
+		it from the disptacher the next time it tries to resume. If <code>thread</code> isn't 
+		registered,	it does nothing.
```

Comment:
the indentation is of here. Also the `addthread` docs say that it returns a coroutine. So best to say here that it takes a coroutine (and not a thread). But that last is a bit of a nitpick.

Reply:
I left it as 'thread' because that is the actual Lua type, and it matches the name of the functions. But I can change the text to say 'coroutine' to be self consistent.